<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginStudentTable</class>
 <widget class="QDialog" name="CLoginStudentTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>当前登录学员</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;宋体&quot;;</string>
     </property>
     <property name="title">
      <string>学员整体管理</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>全体学员当前模式及状态：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_curstate">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;宋体&quot;;</string>
          </property>
          <property name="text">
           <string>    培训模式切换    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;宋体&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>请选择</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>自由模式</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>全体培训</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>全体考核</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>竞赛模式</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonLogoutAll">
        <property name="text">
         <string>踢出所有学员登录</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonExwait">
        <property name="text">
         <string>任务准备（不可开始，等待工况故障下发）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonExamGo">
        <property name="text">
         <string>任务可开始</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="viewLoginStudent">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;宋体&quot;;
</string>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonTask">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>设置任务</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonScene">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>设置工况</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFault">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>设置故障</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRefresh">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>刷   新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>退   出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonTask</tabstop>
  <tabstop>buttonExit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>CLoginStudentTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
